client/app/telecom/pack/xdsl/modem/templates/config/pack-xdsl-modem-templates-config.controller.js

```diff
@@ -0,0 +1,261 @@
+angular.module('managerApp').controller('XdslModemTemplateConfigCtrl', class XdslModemTemplateConfigCtrl {
+  /* @ngInject */
+  constructor($q, $stateParams, $translate, $uibModal, OvhApiXdsl, TucToast, TucToastError) {
+    this.$q = $q;
+    this.$stateParams = $stateParams;
```

Comment:
Not used, so you can get rid of it.

Reply:
Nope. It's used into the fetchTemplates method

Reply:
line 21

Reply:
I don't see it either

Reply:
line 35: .then(names => this.$q.all(_.map(...